x-pack/test/api_integration/deployment_agnostic/apis/observability/apm/service_groups/service_group_count/service_group_count.spec.ts

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import expect from '@kbn/expect';
+import { ApmSynthtraceEsClient } from '@kbn/apm-synthtrace';
+import { DeploymentAgnosticFtrProviderContext } from '../../../../../ftr_provider_context';
+import {
+  createServiceGroupApi,
+  deleteAllServiceGroups,
+  getServiceGroupCounts,
+} from '../service_groups_api_methods';
+import { generateData } from './generate_data';
+
+export default function ApiTest({ getService }: DeploymentAgnosticFtrProviderContext) {
+  const synthtrace = getService('synthtrace');
+  const apmApiClient = getService('apmApi');
+  const start = Date.now() - 24 * 60 * 60 * 1000;
+  const end = Date.now();
+
+  // FLAKY: https://github.com/elastic/kibana/issues/177655
+  describe('Service group counts', () => {
```

Comment:
nit: it is ok to remove flaky comment because ci-bot creates issue based on file path and it will change